bundles/org.openhab.binding.solarwatt/README.md

```diff
@@ -0,0 +1,155 @@
+# Solarwatt Binding
+
+Binding to query a [solarwatt](https://www.solarwatt.de/) [energy manager](https://www.solarwatt.de/energie-management/energymanager) and read the values of all attached devices.
+
+All supported values and devices were discovered while playing with my own energy manager.
+
+
+## Supported Things
+
+* EnergyManager  
+  Location  
+  Forecast  
+  GridFlow  
+  ProfileApp  
+  ScheduleApp  
+  SimpleSwitcher (seems deprecated)  
+  SmartEnergyManagement (seems deprecated)
+
+* BatteryConverter
+  
+* MyReserve  
+  MyReserveInverter  
+  MyReservePowerMeter
+  
+* EVStation (eg. Keba)
+* S0Counter
+* SunSpecInverter (eg. Fronius)  
+
+
+## Discovery
+
+You have to enter the hostname or ip-address of the energymanager itself. The attached devices and supported channels
```

Comment:
```suggestion
You have to enter the hostname or ip-address of the energymanager itself.
The attached devices and supported channels
```
place all sentences on their own new line. Makes future changes cleaner when a line is edited.

Reply:
Put sentence on a new line.